alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -156,5 +162,49 @@ class AlertingSettings {
             -1L,
             Setting.Property.NodeScope, Setting.Property.Dynamic
         )
+
+        val TOTAL_MAX_ACTIONS_PER_TRIGGER = Setting.intSetting(
+            "plugins.alerting.max_actions_per_trigger",
+            DEFAULT_TOTAL_MAX_ACTIONS_PER_TRIGGER,
+            -1, MaxActionsPerTriggersValidator(),
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+        val TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS = Setting.intSetting(
+            "plugins.alerting.total_max_actions_across_triggers",
+            DEFAULT_TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS,
+            -1, TotalMaxActionsAcrossTriggersValidator(),
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+
+        internal class TotalMaxActionsAcrossTriggersValidator : Setting.Validator<Int> {
+            override fun validate(value: Int) {}
+
+            override fun validate(value: Int, settings: Map<Setting<*>, Any>) {}
```

Comment:
We need to make sure this setting is still bigger than `TOTAL_MAX_ACTIONS_PER_TRIGGER`